.github/workflows/publish-cloud.yml

```diff
@@ -21,26 +21,28 @@ jobs:
     steps:
       - name: Checkout branch
         uses: actions/checkout@v3
-      - name: get configuration
+        with:
+          token: ${{ secrets.MAINTAIN_TOKEN }}
+      - name: Get Configuration
         id: configuration
         uses: antifree/json-to-variables@v1.0.1
         with: 
             filename: './.github/workflows/config.json'
             prefix: config
       - name: Test permissions
         uses: ./.github/workflows/permissions
-      - name: Install Node.js 14.x
+        with:
+          ENVIRONMENT: ${{ env.config_deployment_environment }}
+          DEPLOYMENT_PERMITTED: ${{ env.config_deployment_permitted }}
+          DEPLOYMENT_REPOSITORY: ${{ github.config_deployment_repository }}
+      - name: Install Node.js 16.x
         uses: actions/setup-node@v2
         with:
-          node-version: 14.x
+          node-version: 16.x
       - name: Get next release branch
         uses: ./.github/workflows/get-next-release-branch
         with:
-          INPUT_VERSION: ${{ github.event.inputs.version }}
-      - name: Checkout next release branch
-        uses: actions/checkout@v3
-        with:
-          ref: ${{ env.RELEASE_BRANCH }}
```

Comment:
Restore

Reply:
Done, I was also seeing that we didn't check out again to master. So I added a new checkout master before the bump version.